test/release/examples/primers/iterators.chpl

```diff
@@ -142,10 +142,10 @@ writeln();
 // When invoked in a forall loop (or semantically equivalent context),
 // iterators that create parallel tasks and assign work to
 // those tasks are required.  These are known as leader/follower
-// iterators in Chapel, and they are a fairly big topic of their
-// own.  See the leaderfollower.chpl test in this directory for
-// an extended example of defining and using leader/follower
-// iterators.
+// or standalone paralllel iterators in Chapel, and they are a fairly
+// big topic of their own.  See the parIters.chpl test in
+// this directory for an extended example of defining and using
+// leader/follower and standalone parallel iterators.
```

Comment:
I'd be tempted to change the last two sentences of this paragraph to just "Such iterators are a fairly big topic and are described in detail in the parIters.chpl test which lives in this directory", on the argument that the different types of parallel iterators would be covered in there anyways.

If you chose to leave it as is, there's an extra letter "L" in parallel.